core/server/src/main/java/alluxio/master/file/meta/InodeDirectory.java

```diff
@@ -68,9 +68,10 @@ protected InodeDirectory getThis() {
    * Adds the given inode to the set of children.
    *
    * @param child the inode to add
+   * @return true if inode was added successfully, false otherwise
    */
-  public void addChild(Inode<?> child) {
-    mChildren.add(child);
+  public boolean addChild(Inode<?> child) {
```

Comment:
well, as it is, it is not clear what is the purpose of this change as nothing consumes the boolean returned by addChild (and transitively the boolean returned by add)